schedview/compute/__init__.py

```diff
@@ -0,0 +1,25 @@
+from .astro import convert_evening_date_to_night_of_survey, night_events
+from .camera import LsstCameraFootprintPerimeter
+from .scheduler import (
+    compute_basis_function_reward_at_time,
+    compute_basis_function_rewards,
+    create_example,
+    make_scheduler_summary_df,
+    make_unique_survey_name,
+    replay_visits,
+)
+from .survey import compute_maps, make_survey_reward_df
+
+__all__ = [
```

Comment:
When John reformatted some of rubin_sim, he moved the _all_ ahead of the imports and used a tuple. 
I think this matches the DM guidelines - https://developer.lsst.io/python/style.html#standard-code-order-should-be-followed - so since this a PR with a lot of reformatting to match standards goals, could you please also move the __all__ statements accordingly?

Reply:
I'm moving the `__all__` assignments to the top. Although tuples seem to work in practice, the [official python.org documentation](https://docs.python.org/3/tutorial/modules.html), as well as the examples in PEP8, all say that `__all__` should be a list, so I think we should leave them as lists.